.github/actions/build_push/action.yaml

```diff
@@ -0,0 +1,61 @@
+name: "Build and push"
+description: "build and push container image"
+
+inputs:
+  dir:
+    description: "working direcotry"
+    required: false
+    default: ""
+  container-image:
+    description: "container image"
+    required: true
+  github_token:
+    description: "GitHub Token"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Extract targets
+      shell: bash
+      run: |
+        dir=${{ inputs.dir }}
+        if [ "$dir" = "" ]; then dir=${{inputs.container-image }} ; fi
```

Comment:
for readability

```suggestion
        if [ "$dir" = "" ]; then dir=${{ inputs.container-image }} ; fi
```

Reply:
Fix at dae0f70d53a7158fed7fea616c7995f42630b789.